doxygen/dox/TechnicalNotes.dox

```diff
@@ -32,7 +32,1026 @@
 
 /** \page VFL HDF5 Virtual File Layer
 
-\htmlinclude VFL.html
+\section sec_vfl_intro Introduction
+The HDF5 file format describes how HDF5 data structures and dataset raw data are mapped
+to a linear format address space and the HDF5 library implements that bidirectional mapping
+in terms of an API. However, the HDF5 format specifications do not indicate how the format
+address space is mapped onto storage and HDF (version 5 and earlier) simply mapped the format
```

Comment:
(version 5 and earlier) may become confusing.  I suppose it means HDF4 and HDF5?  Does "simply mapped the format address space directly onto a single file by convention" apply to all HDF5 versions?  If so then it should be ok.